homeassistant/components/energyzero/sensor.py

```diff
@@ -187,3 +239,10 @@ def __init__(
     def native_value(self) -> float | datetime | None:
         """Return the state of the sensor."""
         return self.entity_description.value_fn(self.coordinator.data)
+
+    @property
+    def extra_state_attributes(self) -> dict[str, Any]:
+        """Return the optional state attributes."""
+        if self.entity_description.prices_fn is not None:
+            return {"prices": self.entity_description.prices_fn(self.coordinator.data)} 
```

Comment:
This isn't allowed, for more details see the original pull request: <https://github.com/home-assistant/core/pull/83886>

Reply:
Prices colletion is removed!